hw/application_fpga/fw/mta1_mkdf/proto.h

```diff
@@ -35,7 +35,9 @@ enum fwcmd {
 	FW_CMD_RUN_APP		= 0x07,
 	FW_RSP_RUN_APP		= 0x08,
 	FW_CMD_GET_APP_DIGEST	= 0x09,
-	FW_RSP_GET_APP_DIGEST	= 0x10
+	FW_RSP_GET_APP_DIGEST	= 0x10,
```

Comment:
@mchack-work oh, did you see this btw, we skipped 0x0a et al. I guess changing `0x10` is a bad idea. But should we squeeze in the `*_LOAD_USS` starting at 0x0a? Or just ignore the gap... though it annoys me :)